src/twisted/web/_flatten.py

```diff
@@ -229,50 +229,51 @@ def keepGoing(newRoot, dataEscaper=dataEscaper,
         write(b'-->')
     elif isinstance(root, Tag):
         slotData.append(root.slotData)
+        
```

Comment:
Minor comment
Do we need this change?
I know that you might not like the spacing..but other dev might not like your style...so I don't want to start an edit war :)